Useful tools & utilities/In-house libraries.md

```diff
@@ -0,0 +1,37 @@
+
+Often, there will be problems experienced by more than one person in the team or some code which is shared between multiple projects. Those present an excellent opportunity to form a common solution into an open-source library, helping anyone that could make use of it, inside or outside of the company. When creating a library, we strive to make it straightforward, easy-to-use and applicable to everyone, rather than limiting it to our use-cases. 
+
+Developing a library is like working on your mini-project. It can provide you with a sense of pride and accomplishment, as well as being a great learning opportunity. We encourage anyone with ideas to pitch them to our librarian [@JonatanPlesko](https://github.com/JonatanPlesko).
```

Comment:
```suggestion
Developing a library is like working on your mini-project. It can provide you with a sense of pride and accomplishment, as well as being a great learning opportunity.
```
The idea pitching part is strictly internal, so I'd leave it out. Would be weird to be getting random people from other companies messaging me on Github with their library ideas 😄